app/src/main/res/layout/fragment_settings.xml

```diff
@@ -1,14 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:layout_margin="24dp"
+    android:orientation="vertical">
 
-    <TextView
+    <LinearLayout
+        android:id="@+id/export_stopwatches_button"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:gravity="center"
-        android:text="settings"
-        android:textColor="#000"
-        android:textSize="22sp"
-        android:textStyle="bold|italic" />
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Export stopwatches data"
+            android:textColor="@color/black"
+            android:textSize="18dp" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:text="Transfer data to CSV" />
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/export_timestamps_button"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Export timestamps data"
```

Comment:
Should be "Export preferences" with possibility to choose what we want to export. Could be in next version. Someting like:

- [ ] Stopwatches

- [ ] Categories

- [ ] Timestamps

Stopwatch V (chooser of stopwatch to export)